website/client-old/js/controllers/inventoryCtrl.js

```diff
@@ -383,21 +382,19 @@ habitrpg.controller("InventoryCtrl",
     };
 
     $scope.shouldShowPremiumMountRow = function (potion) {
-      potion = Content.premiumHatchingPotions[potion].key;
       var pets = Object.keys(user.items.pets);
-      return pets.find(function (pet) {
-        return pet.indexOf(potion) !== -1;
+      return pets.find(function (petKey) {
```

Comment:
Does it need to be `$scope.functionName`, or should it be `var functionName` ? Sorry for the noob question, but I don't quite get the difference yet.